classifier/__init__.py

```diff
@@ -876,6 +905,8 @@ def audience(cls, identifier, name):
             return cls.AUDIENCE_ADULT
         elif identifier == 'adults only':
             return cls.AUDIENCE_ADULTS_ONLY
+        elif identifier == 'all ages':
+            return cls.AUDIENCE_ALL_AGES
```

Comment:
FreeformAudienceClassifier doesn't seem to have its own test suite. It's possible it got lost when Adrianna and I refactored the classifier code. We can make a new one based on test_age.py.